default/tests/shared/test_annotation.py

```diff
@@ -125,69 +125,69 @@ def test_update_sequence_id_in_cache_list(self):
 
```

Comment:
Nothing has actually changed in this file, I just re-ordered the tests so that test cases for the same methods are grouped together